### PR TITLE
Add bank account number validation for Banco do Brasil and Bradesco

### PR DESCRIPTION
**Pull request by jpaulohe4rt**

### CODE REVIEW

File: src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBBValidator.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import com.github.darkroomdevs.checkdigit.util.ModuloUtil;
+import com.github.darkroomdevs.checkdigit.validator.DigitValidator;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * This class checks whether a specific string represents a valid bank account number of Banco Santander.
+ * <p/>
+ * Reference: <a href="http://hostmarcasite.com.br/fluir/vendor/eduardokum/laravel-boleto/manuais/Regras%20Validacao%20Conta%20Corrente%20VI_EPS.pdf">Banco do Brasil.</a>
+ */
+public final class BankAgencyBBValidator implements DigitValidator {
```
Comment (alfoltran) [inline src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBBValidator.java]: You must use the prefix `BankBranch` when referring to the bank branch number.

File: src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBradescoValidator.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import com.github.darkroomdevs.checkdigit.util.ModuloUtil;
+import com.github.darkroomdevs.checkdigit.validator.DigitValidator;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * This class checks whether a specific string represents a valid bank account number of Banco Bradesco
+ * <p/>
+ * Reference: <a href="https://banco.bradesco/assets/pessoajuridica/pdf/mpo_arquivos_layout_400P.pdf">Banco Bradesco Reference</a>
+ */
+public final class BankAgencyBradescoValidator implements DigitValidator {
```
Comment (alfoltran) [inline src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBradescoValidator.java]: You must use the prefix `BankBranch` when referring to the bank branch number.

File: src/test/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBBValidatorTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
+package com.github.darkroomdevs.checkdigit.validator.brazil;
+
+import com.github.darkroomdevs.checkdigit.validator.DigitValidator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class BankAgencyBBValidatorTest {
```
Comment (alfoltran) [inline src/test/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBBValidatorTest.java]: You must use the prefix `BankBranch` when referring to the bank branch number.

File: src/test/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBradescoValidatorTest.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package com.github.darkroomdevs.checkdigit.validator.brazil;
+
+import com.github.darkroomdevs.checkdigit.validator.DigitValidator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class BankAgencyBradescoValidatorTest {
```
Comment (alfoltran) [inline src/test/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBradescoValidatorTest.java]: You must use the prefix `BankBranch` when referring to the bank branch number.

**Review by alfoltran — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBBValidator.java (hunk 1 above)
Comment (jpaulohe4rt) [reply src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAgencyBBValidator.java]: Alright.

**Review by alfoltran — CHANGES_REQUESTED** [review]
Replace all occurrences of `agency` with `branch`.

**Review by alfoltran — APPROVED** [review]

**Pull request merged**